src/Pages/minjoo/Login/Login.js

```diff
@@ -1,13 +1,200 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import instalogo from "../../../images/minjoo/instagram-logo-1.png";
+import facebook from "../../../images/minjoo/facebook.svg";
+import "./Login.scss";
 
 class Login extends Component {
-    render() {
-        return (
-            <div>
-                
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      password: "",
+      buttonColor: "#c6defa",
+      buttonState: true,
+    };
+  }
+
+  goToMain = () => {
+    // fetch("http://10.58.0.237:8000/user", {
+    //   method: "POST",
+    //   body: JSON.stringify({
+    //     // full_name: "",
+    //     email: this.state.id,
+    //     // phone_number: "",
+    //     // username: "",
+    //     password: this.state.password,
+    //     nickname: "chio",
+    //     name: "치오",
+    //     phonenumber: "13579",
+    //   }),
+    // })
+    //   .then(response => response.json())
+    //   .then(result => console.log("결과: ", result));
+    this.props.history.push("/main-minjoo");
+  };
+
+  handleIdInput = event => {
+    this.setState({
+      id: event.target.value,
+    });
+  };
+
+  handlePwInput = event => {
+    this.setState({
+      password: event.target.value,
+    });
+    console.log(this.state);
+  };
```

Comment:
여러개의 input을 제어할때 하기와 같이 코드를 변경하고 `input` 태그안에 name 속성을 추가해서 가능합니다.
참고 : https://hzoou.tistory.com/216
```suggestion
  handleIdPwInput = (event) => {
    const { name, value } = event.target;
    this.setState({
      [name]: value,
    });
  };
```

Reply:
굿! 이렇게 사용하기 위해서 각 Input 태그 안에 name 속성을 추가해주셔야 해요~!
확인해보시고 적용해보시면 좋을 거 같아요.